gradle/wrapper/gradle-wrapper.properties

```diff
@@ -19,6 +19,6 @@
 
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-bin.zip
```

Comment:
7.2 has been released. Is it easy to use that version instead of 7.1.1?

Reply:
good point, yes that should be easy to upgrade now.